.github/workflows/dcm.yaml

```diff
@@ -14,16 +14,24 @@
 name: Dart Code Metrics
 
 on:
-  pull_request_target
+  push:
+    branches:
+      - master
+  pull_request_target:
+    types: [synchronize, opened, reopened, labeled, unlabeled]
```

Comment:
Could you leave a comment about why these types are important, or perhaps about why others aren't desired?